app/frontend/actions/wallet.ts

```diff
@@ -17,6 +17,8 @@ import loadingActions from './loading'
 import {saveAs} from '../libs/file-saver'
 import {exportWalletSecretDef} from '../wallet/keypass-json'
 
+import {getDefaultTransport} from '../../frontend/helpers/transports'
```

Comment:
this applies only to ledger, so I'd name it "getDefaultLedgerTransportType" and consider keeping the function within the ledger crypto provider and exporting it from there (keeping the "ledger stuff" together), or having a dedicated helper file

Reply:
Refactored